mathesar/imports/csv.py

```diff
@@ -117,7 +117,8 @@ def create_db_table_from_data_file(data_file, name, schema):
     with open(sv_filename, 'rb') as sv_file:
         sv_reader = get_sv_reader(sv_file, header, dialect=dialect)
         column_names = [column_name.strip() for column_name in sv_reader.fieldnames]
-        column_names_alt = [fieldname.strip() if fieldname != constants.ID else constants.ID_ORIGINAL for fieldname in sv_reader.fieldnames]
+        column_names = [f"column_{i}" if column_names[i] == '' else column_names[i] for i in range(len(column_names))]
```

Comment:
Any reason why you are using `range` over the `enumerate` function to loop over column names. `enumerate` seems better for readability.

Reply:
@silentninja  You are absolutely right. I'll fix this right now.